libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_afqp.c

```diff
@@ -651,6 +654,44 @@ TEST( Full_BLE, BLE_Property_WriteLongCharacteristic )
     }
 }
 
+TEST( Full_BLE, BLE_Property_ReadLongCharacteristic )
+{
+    BLETESTreadAttrCallback_t xReadEvent;
+    BTGattResponse_t xGattResponse;
+    BLETESTconfirmCallback_t xConfirmEvent;
+    BTStatus_t xStatus;
+    uint8_t LongReadBuffer[ bletests_LONG_WRITE_LEN ];
+
+    memset( LongReadBuffer, 49, bletests_LONG_WRITE_LEN * sizeof( uint8_t ) );
```

Comment:
Instead of memset you can directly assign to {0} in the declaration above.

Reply:
Right now we just use the same value { '1', '1', '1', ...} that the long write test sent. It's easier to check the value on the RPi side.